### DISCUSSION TITLE
smplayer doesn´t show the lenght fo h264 clip, can´t forward, and overal very slow

### DISCUSSION THREAD

**Discussion by StanislavBriza** [smplayer-dev/smplayer#800] [General]
Hello,
I have got a h.264 video file from home cctv Neos smartcam but when I tried to play, smplayer is just incredibly slow, the video has artefacts (not sure if they are not already in the video, can´t say), but surely I can´t see the total length of the video, I can skip to different time, can´t forward, nothing. It is also very slow, however it plays the video. I have no idea what is going on, any clues? Thank you.

**Comment by monotek505050** [comment]
Below steps may help you on this;

Freshen up SMPlayer by ensuring it's updated to the latest version.
Spruce up your system's codec collection with packs like K-Lite or CCCP to ensure smooth decoding of H.264 videos.
Give your system a spa treatment by checking resource usage and shutting down any unnecessary applications that might be hogging the limelight.
Fine-tune your playback experience by adjusting settings like hardware acceleration in SMPlayer.
Take your video output for a spin by experimenting with different drivers in SMPlayer's settings menu.
Perform a health check on your video files to ensure they're not secretly hiding any corruption issues.
Hit the reset button on SMPlayer's preferences to give it a fresh outlook on life.
Consider inviting alternative players like VLC or MPC-HC to the screening if SMPlayer continues to be camera-shy.
Sprinkle some updates on your operating system and graphics drivers to keep everything running smoothly on the red carpet.
These steps should provide a unique and tailored approach to resolving the issues with SMPlayer.

For more details visit a very good CCTV company here at https://monotek.in/
